modules/createList.js

```diff
@@ -1,19 +1,124 @@
-export default (todo, list) => {
-  list.forEach((item) => {
-    const {
-      description,
-    } = item;
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/213621043-33d8d1ad-11e4-42b9-be01-9bb6e6b36a00.png)

- [ ] You have not set the mode in your webpack.config.js file so because of that this warning has popped up on the screen when your application has been started on the browsers :blush: